src/sentry/tasks/assemble.py

```diff
@@ -799,55 +599,55 @@ def assemble_artifacts(
     # These params have been added for supporting artifact bundles assembling.
     project_ids=None,
     dist=None,
-    upload_as_artifact_bundle=False,
     is_release_bundle_migration=False,
     **kwargs,
-):
+) -> None:
     """
     Creates a release file or artifact bundle from an uploaded bundle given the checksums of its chunks.
     """
     # We want to evaluate the type of assemble task given the input parameters.
-    assemble_task = (
-        AssembleTask.ARTIFACT_BUNDLE if upload_as_artifact_bundle else AssembleTask.RELEASE_BUNDLE
-    )
+    assemble_task = AssembleTask.ARTIFACT_BUNDLE
```

Comment:
The `assemble_task` variable could just be inlined. Also I think the comment above is obsolete now.